### PR TITLE
#3 - Add surface area and volume

### LINKED ISSUE #3: Add more metric notions
See again [this comment](https://github.com/JuliaReach/MathematicalSystems.jl/issues/41).

Proposal. Add these:

- `volume(set)`: returns the volume of the set
- `surface(set)`: returns the surface
 
Again there is some terrain for variations since e.g. the volume of a set such that `dimension` is strictly smaller than its `space_dimension` could be either zero (Lebesgue measure of its ambient space), or not (the Lebesgue measure of the affine hull or relative volume).

### PR DESCRIPTION
**Pull request by mforets**
Closes #3 .

### CODE REVIEW

**Comment by mforets** [comment]
> Maybe again add a test with the Singleton.

👌  I've added tests for these new functions, if you want to check.

File: test/test_sets.jl [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+"""
+    Ball{T} <: AbstractSet
+
+Euclidean ``n``-dimensional ball in ``\\mathbb{R}^n``.
+
+### Fields
+
+- `center` -- vector that represents the center of the sphere
+- `radius` -- scalar that represents the radius of the sphere
```
Comment (schillic) [inline test/test_sets.jl]: `sphere` → `ball`

**Comment by schillic** [comment]
Yes, either use `Project.toml` or add required packages in the Travis build script.

**Comment by mforets** [comment]
Ok, should we prefer one over the other?

**Pull request merged**